### DISCUSSION TITLE
How to correctly compile galmuri?

### DISCUSSION THREAD

**Discussion by TakWolf** [quiple/galmuri#52] [Q&A]
The process is: 

Edit in [bitsnpicas](https://github.com/kreativekorp/bitsnpicas) -> export `.bdf` -> use `mkttf` to `.ttf`  ?

I found some files in https://github.com/quiple/galmuri/tree/main/files, like `xxx.fea`. How to use these files?

I am not very familiar with bitsnpicas, and it can only export bitmap fonts (even OpenType is bitmap format and does not have advanced features). It `.kbitx` source file looks quite simple, with only bitmap data.

<img width="864" alt="image" src="https://github.com/quiple/galmuri/assets/6064962/8d756cf9-f1ab-48a7-b0ab-4d93d1d852db">

**Comment by quiple** [comment]
I rewrite fonts in AFDKO for kerning. This is where the .fea file comes in.

I also use mkttf to generate TrueType fonts with bitmap strikes (which Bits'N'Picas does not support). (Outline TrueType fonts are generated with Bits'N'Picas).

The overall process:

Edit in Bits'N'Picas -> Export `.bdf`, `.ttf`
-> `.bdf` -> Convert to `.ttf` by using mkttf (ex. `Galmuri11Bitmap.ttf`)
-> `.ttf` -> Rewrite `.ttf` by using AFDKO (ex. `Galmuri11.ttf`) -> Compress to `.woff2` by using fontTools
